<?xml version="1.0" encoding="utf-8"?>
<ui version="4.0">
 <class>FinancePanel</class>
 <widget class="QWidget" name="FinancePanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Finance Management</string>
  </property>

  <!-- Add Income Button -->
  <widget class="QPushButton" name="addIncomeButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Income</string>
   </property>
  </widget>

  <!-- Add Expense Button -->
  <widget class="QPushButton" name="addExpenseButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Expense</string>
   </property>
  </widget>

  <!-- Set Budget Button -->
  <widget class="QPushButton" name="setBudgetButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Budget</string>
   </property>
  </widget>

  <!-- Generate Report Button -->
  <widget class="QPushButton" name="generateReportButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Generate Report</string>
   </property>
  </widget>

  <!-- Income Date Line Edit -->
  <widget class="QLineEdit" name="incomeDateLineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>100</width>
     <height>24</height>
    </rect>
   </property>
  </widget>

  <!-- Income Category Line Edit -->
  <widget class="QLineEdit" name="incomeCategoryLineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>100</width>
     <height>24</height>
    </rect>
   </property>
  </widget>

  <!-- Income Amount Line Edit -->
  <widget class="QLineEdit" name="incomeAmountLineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>100</width>
     <height>24</height>
    </rect>
   </property>
  </widget>

  <!-- Expense Date Line Edit -->
  <widget class="QLineEdit" name="expenseDateLineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>100</width>
     <height>24</height>
    </rect>
   </property>
  </widget>

  <!-- Expense Category Line Edit -->
  <widget class="QLineEdit" name="expenseCategoryLineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>180</y>
     <width>100</width>
     <height>24</height>
    </rect>
   </property>
  </widget>

  <!-- Expense Amount Line Edit -->
  <widget class="QLineEdit" name="expenseAmountLineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>100</width>
     <height>24</height>
    </rect>
   </property>
  </widget>

  <!-- Budget Category Line Edit -->
  <widget class="QLineEdit" name="budgetCategoryLineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>100</width>
     <height>24</height>
    </rect>
   </property>
  </widget>

  <!-- Budget Amount Line Edit -->
  <widget class="QLineEdit" name="budgetAmountLineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>300</y>
     <width>100</width>
     <height>24</height>
    </rect>
   </property>
  </widget>

  <!-- Category ComboBox -->
  <widget class="QComboBox" name="categoryComboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>230</width>
     <height>24</height>
    </rect>
   </property>
  </widget>

  <!-- Total Income Label -->
  <widget class="QLabel" name="totalIncomeLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Total Income: 0</string>
   </property>
  </widget>

  <!-- Total Expense Label -->
  <widget class="QLabel" name="totalExpenseLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Total Expense: 0</string>
   </property>
  </widget>

  <!-- Net Income Label -->
  <widget class="QLabel" name="netIncomeLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Net Income: 0</string>
   </property>
  </widget>

  <!-- Budget Utilization Label -->
  <widget class="QLabel" name="budgetUtilizationLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>300</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Utilization: 0%</string>
   </property>
  </widget>

  <!-- Remaining Budget Label -->
  <widget class="QLabel" name="remainingBudgetLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>320</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Remaining Budget: 0</string>
   </property>
  </widget>

  <!-- Category Summary Label -->
  <widget class="QLabel" name="categorySummaryLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>350</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Category Summary: </string>
   </property>
  </widget>

 </widget>
 <resources/>
 <connections/>
</ui>
